dist/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link href="css/style.css" rel="stylesheet">
+  <title>Document</title>
+</head>
+
+<body class="root">
+  <!-- buttons -->
+  <!-- <button class="btn btn--primary">Action</button>
+  <button class="btn btn--secondary">Action</button>
+  <button class="btn btn--text">Action</button> -->
+  <!-- !buttons -->
+
+  <!-- headings -->
+  <!-- <h1 class="heading heading--1">Heading-1</h1>
+  <h1 class="heading heading--2">Heading-2</h1>
+  <h1 class="heading heading--3">Heading-3</h1>
+  <h1 class="heading heading--4">Heading-4</h1>
+  <h1 class="heading heading--5">Heading-5</h1>
+  <h1 class="heading heading--6">Heading-6</h1> -->
+  <!-- !headings -->
+
+  <!-- body copy -->
+  <!-- <div class="body-copy">
+    <div class="body-copy__typo">
+      <p>
+        Hello, sir. I am See-Threepio, human-cyborg relations.
+        My facilities are at your... Well, really! What's wrong with the Falcon? Hyperdrive.
+        I'll get my people to work on it. Good. You know, that ship saved my life quite a few times.
+        She's the fastest hunk of junk in the galaxy. How's the gas mine? Is it paying off for you?
+        Oh, not as well as I'd like. We're a small outpost and not very self-sufficient.
+        And I've had supply problems of every kind. I've had labor difficulties... What's so funny? You.
+        Listen to you - you sound like a businessman, a responsible leader. Who'd have thought that, huh?
+        You know, seeing you sure brings back a few things. Yeah. Yeah, I'm responsible these days.
+        It's the price you pay for being successful.
+      </p>
+    </div>
+  </div> -->
+  <!-- !body copy -->
+  <div class="card">
+    <div class="card__item card__item--headline">
+      <div class="body-copy">
+        <div class="body-copy__typo body-copy__typo--headline">
+          <span class="headline__date">31.07.2016</span>
```

Comment:
Can you explain the `headline__date` class? Where is the `headline` block? Why `date` is an `element`?